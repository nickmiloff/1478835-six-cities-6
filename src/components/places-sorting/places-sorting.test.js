import React from 'react';
import {render} from '@testing-library/react';
import {TestStore as Test} from '../test/test';

import PlacesSorting from './places-sorting';

describe(`PlacesSorting component test`, () => {
  it(`PlacesSorting component' should render correctly`, () => {
    const ACTIVE_OPTION = `Popular`;

    const {container} = render(
        <Test>
          <PlacesSorting activeOption={ACTIVE_OPTION} onChangeOption={() => {}} />
        </Test>
    );

    expect(container).toMatchSnapshot();
  });
});
