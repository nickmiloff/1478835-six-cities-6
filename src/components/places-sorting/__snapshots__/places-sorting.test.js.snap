// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlacesSorting component test PlacesSorting component' should render correctly 1`] = `
<div>
  <form
    action="#"
    class="places__sorting"
    method="get"
  >
    <span
      class="places__sorting-caption"
    >
      Sort by
    </span>
    <span
      class="places__sorting-type"
      tabindex="0"
    >
      Popular
      <svg
        class="places__sorting-arrow"
        height="4"
        width="7"
      >
        <use
          xlink:href="#icon-arrow-select"
        />
      </svg>
    </span>
    <ul
      class="places__options places__options--custom"
    >
      <li
        class="places__option places__option--active\\""
        tabindex="0"
      >
        Popular
      </li>
      <li
        class="places__option"
        tabindex="0"
      >
        Price: low to high
      </li>
      <li
        class="places__option"
        tabindex="0"
      >
        Price: high to low
      </li>
      <li
        class="places__option"
        tabindex="0"
      >
        Top rated first
      </li>
    </ul>
  </form>
</div>
`;
